pdcupdater/handlers/modules.py

```diff
@@ -130,12 +136,10 @@ def get_or_create_unreleased_variant(self, pdc, body):
         # version/release, but for now we just do the right mapping here...
         variant_version =  body['stream'] # This is supposed to be equal to version
         variant_release =  body['version'] # This is supposed to be equal to release
+        variant_uid = "%s-%s-%s" % (variant_id, variant_version, variant_release)
```

Comment:
Not a deal breaker, but I like the way this is done in `create_unreleased` :) :
variant_uid = "{n}-{v}-{r}".format(n=name, v=version, r=release)